crates/torii/core/src/engine.rs

```diff
@@ -138,7 +138,6 @@ impl<P: Provider + Sync> Engine<P> {
         if from < latest_block_number {
             // if `from` == 0, then the block may or may not be processed yet.
             let from = if from == 0 { from } else { from + 1 };
-            info!(target: LOG_TARGET, from = %from, to = %latest_block_number, "Syncing range.");
```

Comment:
May be nice to keep this in `debug!`, wdyt?

Reply:
agree